KarhooUISDK/Workers/VehicleRules/VehicleRulesProvider.swift

```diff
@@ -79,13 +81,22 @@ final class KarhooVehicleRulesProvider: VehicleRulesProvider {
                     }
                 return rule
             }
+            
             let specificRule = vehicleRules?.rules.first { rule in
                 guard rule.ruleType == .specific else { return false }
                 let ruleTags = Set(rule.tags.map { $0.lowercased() })
                 let quoteVehicleTags = Set(quote.vehicle.tags.map { $0.lowercased() })
                 return rule.type.lowercased() == quote.vehicle.type.lowercased() && ruleTags == quoteVehicleTags
             }
-            let rule = (specificRule ?? anyMatchingTagRule) ?? (defaultTypeRule ?? fallbackRule)
+            
+            var rule: VehicleImageRule? {
+                if quote.vehicle.type.lowercased() == VehicleType.moto.rawValue.lowercased() {
```

Comment:
What are your thoughts on replacing this condition with something like:
`if VehicleType(rawValue: quote.vehicle.type.uppercased()) == .moto {`

I know it's essentially the same, but I feel uneasy comparing the string values :D